docs/CONFIG-VARS.md

```diff
@@ -168,6 +168,7 @@ When setting V4_CFG_MANAGE_STORAGE to true, A new storage classes will be create
 | V4M_KIBANA_CERT | Path to tls certificate to use for dashboards ingress | string |<V4M_CERT> | false | If both this and V4M_CERT are not set a self-signed cert will be used | cluster-logging |
 | V4M_KIBANA_KEY | Path to tls key to use for dashboards ingress | string | <V4M_KEY> | false | If both this and V4M_KEY are not set a self-signed cert will be used | cluster-logging |
 | V4M_KIBANA_PASSWORD | Dashboards admin password | string | randomly generated | false | If not provided, a random password will be generated and written to the log output | cluster-logging |
+| V4M_KIBANA_LOGADM_PASSWORD | Dashboards logadm user's password | string | randomly generated | false | If not provided, a random password will be generated and written to the log output | cluster-logging |
```

Comment:
Consider changing to "If not provided and V4M_KIBANA_PASSWORD is not set, a random password..."

Also regarding Scenario #3 in your tests, to mimic the behavior of the logadm password being set to the admin password if that is provided, would it also follow that we should set the logadm password to the random value assigned to the admin password if neither are set, instead of setting both of them to unique random passwords?

Reply:
Updated the Note.
For scenario 3, if both the logadm and admin passwords are not set, then random passwords are generated for both. The random passwords are expected to be unique.